Assets/HoloToolkit/Boundary/README.md

```diff
@@ -14,23 +14,23 @@ A simple quad scaled up to 10x that will be rendered as the floor for an immersi
 
 #### BoundaryManager.cs
 Places a floor quad to ground the scene.
-Allows you to check if your GameObject is within setup boundary on the immersive headset.
+Allows you to check whether your GameObject is within setup boundary on the immersive headset or not.
 Boundary can be configured via the Mixed Reality Portal.
 
 ### [Tests](Tests)
 To use the scene:
 
-1. Navigate to the Tests folder.
+1. Locate the Tests folder.
 2. Double click on the test scene you wish to explore.
 3. Either click "Play" in the unity editor or File -> Build Settings.
 4. Add Open Scenes, Platform -> Windows Store, SDK -> Universal 10, Build Type -> D3D, Check 'Unity C# Projects'.
 5. Click 'Build' and create an App folder. When compile is done, open the solution and deploy to device.
 
 #### BoundaryTest.unity 
-Shows how to check if an object is within boundary and render a floor quad.
+Shows how to check whether an object is within boundary and render a floor quad.
 
 We render the floor quad at (0,0,-3) in editor.
-There are 4 different cubes in the test scene which try to demonstrate if an object is within or outside the setup boundary.
+There are 4 identical cubes with different position in the test scene which try to demonstrate if an object is within or outside the setup boundary.
```

Comment:
These kinds of changes are good.

Reply:
So this won't need to be revert right haha, hope that everything will be approved and just let me know if there's anything that I need to change.

Reply:
So can this be approve?